src/Compilers/CSharp/Portable/Binder/Binder_Lambda.cs

```diff
@@ -237,6 +243,32 @@ static void checkAttributes(AnonymousFunctionExpressionSyntax syntax, SyntaxList
             }
         }
 
+        private (RefKind, TypeWithAnnotations) BindExplicitLambdaReturnType(TypeSyntax syntax, BindingDiagnosticBag diagnostics)
+        {
+            MessageID.IDS_FeatureLambdaReturnType.CheckFeatureAvailability(diagnostics, syntax);
+
+            RefKind refKind = RefKind.None;
+            if (syntax is RefTypeSyntax refTypeSyntax)
```

Comment:
There's a helper that might be re-used here:  `syntax = syntax.ReturnType.SkipRef(out RefKind refKind);`

Reply:
Updated, thanks.